src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -329,10 +345,38 @@ def load_emr_ihd(key: str, location: str) -> pd.DataFrame:
     map = {
         data_keys.MYOCARDIAL_INFARCTION.EMR_ACUTE: 24694,
         data_keys.MYOCARDIAL_INFARCTION.EMR_POST: 15755,
+        data_keys.ANGINA.EMR: 1817,
     }
     return _load_em_from_meid(location, map[key], "Excess mortality rate")
 
 
+def load_csmr_all_zeros(emr_source: str, location: str) -> pd.DataFrame:
+    # We cannot query sequela for CSMR. Instead, let's return all zeros since
+    # we need something for the SI model.
+    # 
+    # Note that 100% of CSMR for IHD has been assigned to MI. This then requires
+    # that other IHD causes (angina and heart failure) must be assigned 
+    # zero CSMR or else we would underestimate the IHD mortality rate because
+    # we'd be subtracting off too much CSMR.
+    #
+    # TODO: If desired for validation purposes, we can implement an approach that
+    # calculates the individual IHD cause CSMR like:
+    #   csmr_angina = (
+    #       load_ihd_prevalence(data_keys.IHD.ANGINA_PREV, location) *
+    #       load_ihd_emr(data_keys.IHD.ANGINA_EMR, location) *
+    #       person-time
+    #     )
```

Comment:
This is some great documentation.

Reply:
Rajan and I had to decrypt what had previously been done with is always a good indicator that it should be documented